runtime/vscode/overrides/vscode-file-commands.vim

```diff
@@ -49,10 +49,11 @@ AlterCommand e[dit] Edit
 AlterCommand ex Ex
 AlterCommand ene[w] Enew
 AlterCommand fin[d] Find
-AlterCommand w[rite] Write
 AlterCommand sav[eas] Saveas
 AlterCommand wa[ll] Wall
 AlterCommand q[uit] Quit
+" TODO: Wq/Wqall need to be handled with the autocmd
```

Comment:
Unfortunately, this means `:w` and `:wq` have different behavior. I'm hoping to resolve this before undrafting